packages/schematics/angular/guard/schema.json

```diff
@@ -49,9 +49,13 @@
       "items": {
         "enum": ["CanActivate", "CanActivateChild", "CanDeactivate", "CanLoad", "CanMatch"],
         "type": "string"
```

Comment:
Should implements be deprecated?
If so add an option.
`"x-deprecated: "Use '--type' instead.........."`

Reply:
Looks like this has not been done.

Reply:
I didn't deprecate it in this PR but it's in the one that updates the defaults. That way, this change is purely a feature addition rather than combining a feature, a deprecation, and an update to the default values.